index.js

```diff
@@ -56,15 +54,60 @@ const socketYmlSchema = {
   additionalProperties: false,
 }
 
+/**
+ * @typedef SocketYmlV1
+ * @property {string[]} [ignore]
+ * @property {{ [issueName: string]: boolean }} [issues]
+ * @property {boolean} [beta] unused v1 option
+ * @property {boolean} [enabled] enable/disable the Socket.dev GitHub app entirely
+ * @property {boolean} [projectReportsEnabled] enable/disable Github app project report checks
+ * @property {boolean} [pullRequestAlertsEnabled] enable/disable GitHub app pull request alert checks
+ */
+
+/** @type {import('ajv').JSONSchemaType<SocketYmlV1>} */
+const socketTmlSchemaV1 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  properties: {
+    ignore: {
+      type: 'array',
+      items: { type: 'string' },
+      nullable: true,
+    },
+    issues: {
+      type: 'object',
+      nullable: true,
+      required: [],
+      additionalProperties: { type: 'boolean' },
+    },
+    beta: { type: 'boolean', nullable: true, default: true },
+    enabled: { type: 'boolean', nullable: true, default: true },
+    projectReportsEnabled: { type: 'boolean', nullable: true, default: true },
+    pullRequestAlertsEnabled: { type: 'boolean', nullable: true, default: true },
+  },
+  minProperties: 1,
+  additionalProperties: false,
+}
+
 const ajv = new Ajv({
   allErrors: true,
   coerceTypes: 'array',
   logger: false,
   removeAdditional: 'failing',
+  useDefaults: true
 })
 
 const validate = ajv.compile(socketYmlSchema)
 
+const ajvV1 = new Ajv({
+  allErrors: true,
+  coerceTypes: 'array',
+  logger: false,
+  useDefaults: true
+})
+
```

Comment:
I think you can reuse the same `Ajv` instance, no?

Reply:
It has a hard time differentiating between invalid v2 and a v1 if I don't do this. Try using the same settings and see where it fails.

Reply:
Right, because of the `removeAdditional` in the other one, I get it now :)

How about we name it as the strict one?

Reply:
Ok. I'll rename it